pkg/trace/api/endpoints.go

```diff
@@ -80,4 +80,8 @@ var endpoints = []endpoint{
 		Pattern: "/v0.6/stats",
 		Handler: func(r *HTTPReceiver) http.Handler { return http.HandlerFunc(r.handleStats) },
 	},
+	{
+		Pattern: "/appsec/v0.1/proxy/",
```

Comment:
Can we use `/appsec/v1/input` instead?

In fact, as the agent is only a proxy for now, the `v1/input` suffix defines the "event sending protocol" while the base URL `http://localhost:8126/appsec/` & `https://appsecevts-http-intake.logs.datad0g.com/` are destinations.

The libraries could then send events to the agent or intake directly only by changing the base URL, which is useful for development and makes implementation more straightforward.

Reply:
I'd rather have `/appsec/v0.1/proxy` forwarding to `https://appsecevts-http-intake.logs.datad0g.com` so that the agent API endpoint to use becomes `/appsec/v0.1/proxy/v1/input`

wdyt @gbbr? the base problem comes from the case when a library is in agentless mode. This would simplify a bit the configuration by just having to change the API URL "prefix" part (eg. http://localhost:8126/appsec/v0.1/proxy/v1/input would become https://appsecevts-http-intake.logs.datad0g.com/v1/input)

Reply:
I think `/appsec/v1/input` is the right way to go. We use the same for profiling: `/profiling/v1/input`.

Reply:
by the way, this is why the profiling api endpoint of the trace-agent is `/profiling/v1/input`: to have the same endpoint URL suffix as the intake's

Reply:
I'd also be ok with `/appsec/proxy/v1/input`. That would still hold for the example given by @nizox. I think having a double version in one single URL is a bit much.

Reply:
Done: `/appsec/v0.1/proxy` becomes `/appsec/v1/input`
The v0.1 Agent API versioning is dismissed for now. We'll reintroduce it in the future if needed by an actual agent API.

Reply:
I think we are good to go with this during for the preview version.
There won't be any extra appsec endpoint as far as anyone could tell me.